src/components/monsters/Monster.jsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { Button, CardActions } from '@mui/material';
+import PropTypes from 'prop-types';
+import ImageOfMonster from './Image/ImageOfMonster';
+import s from './Monsters.module.css';
+
+export default function Monster(props) {
+    return (
+        <div className={s.zoom}>
+            <Card className={s.card} sx={{ width: 250 }}>
+                <ImageOfMonster id={props.id} />
+                <CardContent className={s.cardContent}>
+                    <Typography gutterBottom variant="h5" component="div">
```

Comment:
I don't see theme.js, you should learn how to globally work with styles in MUI. How variant works